observability/telemetry/src/main/scala/com/daml/telemetry/OpenTelemetryOwner.scala

```diff
@@ -1,58 +1,69 @@
 // Copyright (c) 2023 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
 // SPDX-License-Identifier: Apache-2.0
 
-package com.daml.metrics
+package com.daml.telemetry
 
 import com.daml.ledger.resources.{Resource, ResourceContext, ResourceOwner}
-import com.daml.metrics.OpenTelemetryMeterOwner.buildProviderWithViews
 import com.daml.metrics.api.MetricHandle.Histogram
 import com.daml.metrics.api.opentelemetry.OpenTelemetryTimer
 import com.daml.metrics.api.reporters.MetricsReporter
 import com.daml.metrics.api.reporters.MetricsReporter.Prometheus
-import io.opentelemetry.api.metrics.Meter
+import com.daml.telemetry.OpenTelemetryOwner.addViewsToProvider
+import io.opentelemetry.api.OpenTelemetry
 import io.opentelemetry.exporter.prometheus.PrometheusCollector
+import io.opentelemetry.sdk.autoconfigure.AutoConfiguredOpenTelemetrySdk
+import io.opentelemetry.sdk.metrics.SdkMeterProviderBuilder
 import io.opentelemetry.sdk.metrics.common.InstrumentType
 import io.opentelemetry.sdk.metrics.view.{Aggregation, InstrumentSelector, View}
-import io.opentelemetry.sdk.metrics.{SdkMeterProvider, SdkMeterProviderBuilder}
 
 import scala.annotation.nowarn
 import scala.concurrent.Future
 import scala.jdk.CollectionConverters.SeqHasAsJava
 
 @nowarn("msg=deprecated")
-case class OpenTelemetryMeterOwner(enabled: Boolean, reporter: Option[MetricsReporter])
-    extends ResourceOwner[Meter] {
+case class OpenTelemetryOwner(setAsGlobal: Boolean, reporter: Option[MetricsReporter])
```

Comment:
`setAsGlobal` is not used in this class, or in the whole project.
It feels like it should be used for `setResultAsGlobal`.

Reply:
And the mystery of failing tests is solved :face_with_head_bandage: